README.md

```diff
@@ -4,23 +4,8 @@ Module to work with an IBM Cloud Object Storage instance. If the `provision` fla
 
 **Note:** This module follows the Terraform conventions regarding how provider configuration is defined within the Terraform template and passed into the module - https://www.terraform.io/docs/language/modules/develop/providers.html. The default provider configuration flows through to the module. If different configuration is required for a module, it can be explicitly passed in the `providers` block of the module - https://www.terraform.io/docs/language/modules/develop/providers.html#passing-providers-explicitly.
```

Comment:
Is it still needed? These conventions are pretty much mainstream at this stage, so it may be safe to assume consumer is aware of those?